Store additional headers and query parameters in the secret (#4341)
kyma-project/kyma: docs/application-connector/08-04-register-secured-api.md

        }
```

### Use headers and query parameters for custom authentication

You can specify additional headers and query parameters that will be injected to requests to the target API.

> Headers and query parameters are both stored in the Kubernetes Secret, therefor they can be used for custom authentication methods.  

Reviewer: ```suggestion
The Kubernetes Secret stores headers and query parameters which you can use for custom authentication methods.  
```
